lib/oli_web/controllers/resource_controller.ex

```diff
@@ -52,8 +52,20 @@ defmodule OliWeb.ResourceController do
       nil ->
         render_not_found(conn, project_slug)
 
+      %{content: %{"advancedDelivery" => true}} = revision ->
+        put_root_layout(conn, {OliWeb.LayoutView, "chromeless.html"})
```

Comment:
I suppose that preview mode should also use the "displayApplicationChrome" flag for this?